CodeGeneration/Sources/SyntaxSupport/DeclNodes.swift

```diff
@@ -1996,6 +1996,11 @@ public let DECL_NODES: [Node] = [
     parserFunction: "parseSourceFile",
     traits: ["WithStatements"],
     children: [
+      Child(
+        name: "Hashbang",
+        kind: .token(choices: [.token(tokenKind: "IdentifierToken")]),
```

Comment:
This would need to be `HashbangToken`, right? Because you’re not parsing this as an `IdentifierToken`. This should fail if you compile swift-syntax with `SWIFTSYNTAX_ENABLE_RAWSYNTAX_VALIDATION`.